firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -48,8 +48,7 @@ android {
 }
 
 dependencies {
-  api("com.google.firebase:firebase-common:20.4.0")
```

Comment:
We would need dependency on both firebase-common and firebase-common-ktx. This will prevent issues wrt collision when used with the old version of firebase-common-ktx